.gitlab-ci.yml

```diff
@@ -49,7 +49,7 @@ run-tests:
   stage: build
   script:
     - cd tools/docker/builder/openwrt
-    - ./build.sh targets/$TARGET.args 2>&1 | tee build.log | grep -E '^make\[[12]\]|^Step' --color=never --line-buffered
+    - ./build.sh -v -d "$TARGET" -t "prplmesh-builder-$TARGET:$CI_COMMIT_SHORT_SHA-$CI_PIPELINE_ID" 2>&1 | tee build.log | grep '^make\[[12]\]\|^Step\|^ --->' --color=never --line-buffered
```

Comment:
I think its better to change `$TARGET` to `$TARGET_DEVICE` since it will be more readable this way IMO.

Reply:
Done in a separate commit, since the variables are already in master, and the changes in this PR are not related.